docs/core-services/sqlite-db-service.md

```diff
@@ -60,7 +60,41 @@ The SQLite DB provides the following configuration parameters:
 
 ### Selecting a database instance for existing components
 
-TBD
+A database instance is identified by its **Kura service PID**. The PID for instances created using the Web UI is the string entered in the Name field at step 2 of the previous section.
+
+The built-in components that use database functionalities allow to specify which instance to use in their configuration. These components are the **DataService** component of the cloud stack, the **Wire Record Store** and **Wire Record Query** wire components. The configuration of each component contains a property that allows to specify the Kura Service PID of the desired instance.
+
+!!! warning
+    Using SQLite database instances through the deprecated **DbFilter** and **DbStore** components is not supported.
+
+### Usage through Wires
+
+It is possible to store and extract Wire Records into/from a SQLite database instance using the **Wire Record Store** and **Wire Record Query** wire components.
+
+When a Wire Record is received by a **Wire Record Store** attached to a SQLite based database instance, the data will be stored in a table whose name is current value of the **Record Collection Name** configuration parameter of the Wire Component.
+
+Each property contained in a Wire Record will be appended to a column with the same name as the property key, a new column will be created if it not already exists.
```

Comment:
> name as the property key, a new column will be created if it not already exists.

name as the property key. A new column will be created if it not already exists.